packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -4086,10 +4201,15 @@ describe('plugin-meetings', () => {
     describe('#enableMusicMode', () => {
       beforeEach(() => {
         meeting.isMultistream = true;
-          meeting.mediaProperties.webrtcMediaConnection = {
-            setCodecParameters: sinon.stub().resolves({}),
-            deleteCodecParameters: sinon.stub().resolves({}),
-          };
+        meeting.sendSlotManager = new SendSlotManager({logger: { info: () => {}}});
+        meeting.sendSlotManager.createSlot({
```

Comment:
again we're creating a send slot - see my previous comment about making sure this doesn't impact other tests

Reply:
Addressed by making changes to this and responding to previous comment as well